<launch>
<arg name="dynreconfigure_name" />   
<node pkg="parameter_tuning" type="parameter_tuner.py" name="px4_parameter_tuner" output="screen">
        
	<param name="dynreconfigure_name" type="string" value="$(arg dynreconfigure_name)" />
	<!--
	<param name="device" type="string" value="cpu" />
	-->
  </node>
</launch>
